guides/source/active_support_core_extensions.md

```diff
@@ -1703,6 +1703,20 @@ The method `parameterize` normalizes its receiver in a way that can be used in p
 "Kurt Gödel".parameterize # => "kurt-godel"
 ```
 
+To preserve the case of the string, set the `preserve_case` argument to true. By default, `preserve_case` is set to false.
+
+```ruby
+"John Smith".parameterize(preserve_case: true) # => "John-Smith"
+"Kurt Gödel".parameterize(preserve_case: true) # => "Kurt-Godel"
+```
+
+To use a custom separator, override the `separator` argument.
+
+```ruby
+"John Smith".parameterize(separator: "_") # => "john\_smith"
+"Kurt Gödel".parameterize(separator: "_") # => "kurt\_godel"
```

Comment:
Interesting, did you check the output in the rendered guides (you can generate them by running `rake guides:generate` inside the guides directory)? If it is still rendered incorrectly it seems like a bug in our set up :cry:

(But if you confirmed that it is indeed necessary, it's fine to leave it here, we can fix the guides generator bug another day)